archive.istio.io/build.sh

```diff
@@ -28,23 +28,27 @@ else
   cd $GITDIR
 fi
 
-FILESTOPATCH=(_includes/nav.html index.html)
+FILESTOPATCH=(_includes/nav.html _includes/header.html index.html)
 VERSIONS_LIST=""
 for rel in "${TOBUILD[@]}"
 do
   NAME=$(echo $rel | cut -d : -f 1)
   TAG=$(echo $rel | cut -d : -f 2)
 	echo "Building '$NAME' from $TAG"
+  git checkout -- .
+  git clean -f
   git checkout $TAG
+  git pull 2> /dev/null
```

Comment:
so for master or a branch it gets updated, the 2> /dev/null is to hide error when pull'ing a sha